centreon/www/class/config-generate/AdditionalConnectorVmWareV6.class.php

```diff
@@ -101,4 +101,30 @@ public function generateFromPollerId(int $pollerId): void
     {
         $this->generate($pollerId);
     }
+
+    /**
+     * @param $dir
+     *
+     * @throws \RuntimeException|\Exception
+     */
+    protected function writeFile($dir)
```

Comment:
Although the method describes its intention, it nevertheless lacks context.
What type of file are you creating?